app.html

```diff
@@ -1,22 +1,41 @@
-<!doctype html>
+<!DOCTYPE html>
 
 <html lang="en">
-
-<head>
-    <meta charset="utf-8">
+  <head>
+    <meta charset="utf-8" />
     <title>Contacts List</title>
-    <link rel="stylesheet" href="app.css">
-</head>
+    <link rel="stylesheet" href="app.css" />
+  </head>
 
-<body>
-    <div id="app">
-        <h1>Aaron's Contact List</h1>
-        <div id="contact-list-container">
-            Placeholder for contact list container
-            <div class="search-bar-container"></div>
-        </div>
-        <script src="js/scripts.js"></script>
-    </div>
-</body>
+  <body>
+    <div id="app" class="float-container">
```

Comment:
why are we calling the class "float-container"?

Reply:
I removed these and utilize Flex-Box